EIPS/eip-compliant.md

```diff
@@ -0,0 +1,198 @@
+---
+eip: <to be assigned>
+title: Compliant Token
```

Comment:
I suggest to called it RegulatedToken. Isnt it so do the harbour naming? ( r-token)